docs/dex-v2-worker.md

```diff
@@ -0,0 +1,44 @@
+# LBE V2 Documentation
```

Comment:
1. move the content to `transaction.md` file
2. change the filename to `dex-transaction.md`
3. Change the Header to `Minswap DEX Classes documentation`